lib/pengine/container.c

```diff
@@ -978,8 +1165,9 @@ container_print(resource_t * rsc, const char *pre_text, long options, void *prin
         pre_text = " ";
     }
 
-    status_print("%sDocker container%s: %s [%s]%s%s\n",
-                 pre_text, container_data->replicas>1?" set":"", rsc->id, container_data->image,
+    status_print("%s%s container%s: %s [%s]%s%s\n",
+                 pre_text, container_type_as_string(container_data->type),
+                 container_data->replicas>1?" set":"", rsc->id, container_data->image,
```

Comment:
This makes sense, but it changes the output from "Docker" to "docker" which breaks some regression tests. I'm thinking we can just use "Docker" as CONTAINER_TYPE_DOCKER_S; I don't think anyone's relying on the crm_mon XML output yet.